self_hosting_machinery/webgui/selfhost_queue.py

```diff
@@ -38,4 +38,5 @@ def models_available(self) -> List[str]:
                 # self._models_available.append('gpt3.5')
                 # self._models_available.append('gpt4')
                 self._models_available.append('longthink/stable')
+        self._models_available.append("gte")
```

Comment:
embed model should be in inference settings and more: we want to run it as a model